frontend/packages/metal3-plugin/src/plugin.ts

```diff
@@ -112,6 +112,17 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [FLAGS.BAREMETAL, METAL3_FLAG],
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: ['/k8s/cluster/nodes/'],
+      loader: async () =>
```

Comment:
Nit - isn't it simpler to avoid async/await syntax here?

```ts
loader: () => import( /* .. */ ).then((m) => m.default),
```

Above form is commonly used across other plugin entry modules.

Reply:
Done